packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -199,17 +211,50 @@ export function getDefaultProvider(input: FieldProvider): DefaultProvider {
 	return input as unknown as DefaultProvider;
 }
 
+/**
+ * Default field schema metadata shape.
```

Comment:
Since this is public, maybe a link to a more central type (FieldSchema) to help flesh this summary out?